src/actions/actions.py

```diff
@@ -12,7 +12,7 @@
 from rasa_sdk.executor import CollectingDispatcher
 import firebase_admin
 from firebase_admin import credentials, firestore, auth
-cred = credentials.Certificate("acs-upb-mobile-dev-firebase-adminsdk-mgl5c-9617ffc92e.json")
+cred = credentials.Certificate("acs-upb-mobile-dev-firebase-adminsdk-mgl5c-d8e16c2d02.json")
```

Comment:
I think we can rename json file - acs-key or something like that and add this rename to the contributing file.  Gitignore must also modified with the new key's name

Reply:
It should be renamed so that we all have the same key's name